rosbag2_py/test/test_sequential_writer.py

```diff
@@ -0,0 +1,103 @@
+# Copyright 2020 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import glob
+import os
+import time
+import shutil
+
+from rclpy.serialization import deserialize_message, serialize_message
+import rosbag2_py._rosbag2_py as rosbag2_py
```

Comment:
From a user perspective, I think it would be best if they didn't have to import the submodule `_rosbag2_py`.
I.e. simply this to

```python
import rosbag2_py
```
 
I vaguely recall having an issue achieving this though, but I don't recall.

Reply:
You added a TODO in CMakeLists.txt [line 29](https://github.com/mabelzhang/rosbag2/blob/rosbag2_py/rosbag2_py/CMakeLists.txt#L29) to `Consider modifying ament_python_install_module to accept targets`. Does something need to be added to ament in order to do that?

Reply:
I don't think it's related? I guess I was thinking that TODO would be better than having to install the module manually here: https://github.com/mabelzhang/rosbag2/blob/d261c63ef4a1ec1e52cf768c58667a5b8d598284/rosbag2_py/CMakeLists.txt#L40-L44

Reply:
ab426d9 simplifies the import. The `__all__` trick in `__init__.py` worked. Simply importing in `__init__.py` didn't (or I couldn't figure out how).